aws-resourceexplorer2-defaultviewassociation/src/main/java/software/amazon/resourceexplorer2/defaultviewassociation/CreateHandler.java

```diff
@@ -73,7 +84,9 @@ public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
 
         return ProgressEvent.<ResourceModel, CallbackContext>builder()
                 .resourceModel(model)
-                .status(OperationStatus.SUCCESS)
+                .status(OperationStatus.IN_PROGRESS)
+                .callbackContext(callbackContext)
+                .callbackDelaySeconds(30)
```

Comment:
Do we really want delay to be this long? Or will something shorter do the trick?

Reply:
something shorter should do. Updating